src/keymap.cpp

```diff
@@ -35,6 +35,16 @@ static op_desc opdescs[] = {
 	{ OP_RANDOMUNREAD,		"random-unread",			"^K",	_("Go to a random unread article"),		KM_FEEDLIST | KM_ARTICLELIST | KM_ARTICLE },
 	{ OP_OPENBROWSER_AND_MARK, "open-in-browser-and-mark-read",                   "O",    _("Open article in browser and mark read"),           KM_ARTICLELIST },
 	{ OP_OPENINBROWSER,		"open-in-browser",			"o",	_("Open article in browser"),			KM_FEEDLIST | KM_ARTICLELIST | KM_ARTICLE },
+    {
```

Comment:
From a history standpoint, I'd rather keep this change to one commit. I mean, preserve the formatting as is, and let us change it all at once in #383.

Also, we use tabs for indentation.